Classes/Search/SearchResultSectionController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SearchResultSectionController.swift
+//  Freetime
+//
+//  Created by Sherlock, James on 28/07/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import IGListKit
+
+final class SearchResultSectionController: ListGenericSectionController<SearchResult> {
+
+    private let client: GithubClient
+    
+    init(client: GithubClient) {
+        self.client = client
+        super.init()
+    }
+    
+    override func sizeForItem(at index: Int) -> CGSize {
+        guard let width = collectionContext?.containerSize.width else { fatalError("Missing context") }
```

Comment:
This is so much better than what I previously did (`return .zero` Hah, good luck debugging that you fool!)